common/idrepo/lib/src/main/java/org/apache/syncope/common/lib/types/AuditElements.java

```diff
@@ -33,9 +33,9 @@ public enum EventCategoryType {
         LOGIC("LOGIC"),
         WA("WA"),
         TASK("TASK"),
-        PROPAGATION("PropagationTask"),
-        PULL("PullTask"),
-        PUSH("PushTask"),
+        PROPAGATION("PROPAGATION"),
```

Comment:
since now all enum cases have their `name()` associated, can't we just change this enum as follows?

```java
    public enum EventCategoryType {
        LOGIC,
        WA,
        TASK,
        PROPAGATION,
        PULL,
        PUSH,
        CUSTOM;
    }
```

Reply:
Yes, sure, going to do this.